packages/frameworks/src/frameworks.ts

```diff
@@ -1932,6 +1932,33 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Storybook',
+    slug: 'storybook',
+    logo: 'https://api-frameworks.vercel.sh/framework-logos/storybook.svg',
+    tagline: 'Frontend workshop for UI development',
+    description:
+      'Storybook is a frontend workshop for building UI components and pages in isolation.',
+    website: 'https://storybook.js.org/',
+    ignoreRuntimes: ['@vercel/next', '@vercel/node'],
+    useRuntime: { src: 'package.json', use: '@vercel/static-build' },
+    settings: {
+      installCommand: {
+        placeholder: '`yarn install`, `pnpm install`, or `npm install`',
+      },
+      buildCommand: {
+        placeholder: '`npm run build-storybook`',
+        value: 'npm run build-storybook',
```

Comment:
value should not use `npm run` (see others)

Reply:
Done!

Reply:
@cb1kenobi I still see `npm run build-storybook`

It should be `storybook build`

Also now that I think about it, the placeholder would be `npm run build` since thats the default.

Reply:
Whoops, done!